Page not found following "Learn more about OpenShift update channels" (#29459)
openshift/openshift-docs: modules/understanding-upgrade-channels.adoc

{product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the `openshift-install`
binary file for a specific version of {product-title} always installs that version.

ifndef::openshift-origin[]
{product-title} {product-version} offers the following upgrade channels:

* `candidate-{product-version}`
* `fast-{product-version}`
* `stable-{product-version}`
* `eus-4.6` (only available when running 4.6)

endif::openshift-origin[]
ifdef::openshift-origin[]
{product-title} {product-version} offers the following upgrade channel:

> * `stable-{product-version}`

Reviewer: That would render as `stable-latest` on docs.okd.io, lets hardcode `stable-4`

You can use the `fast-{product-version}` channel to upgrade from a previous minor version of
{product-title}.
endif::openshift-origin[]

[discrete]
== stable-{product-version} channel
ifndef::openshift-origin[]
While the `fast-{product-version}` channel contains releases as soon
as their errata are published, releases are added to the `stable-{product-version}` channel
after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services,
and pre-production and production environments that participate in connected customer program about the stability of the release.
endif::openshift-origin[]
ifdef::openshift-origin[]

> Releases are added to the `stable-{product-version}` channel

Reviewer: ```suggestion
Releases are added to the `stable-4` channel
```

> after a delay. During this delay, data is collected from Red Hat SRE teams, Red Hat support services,

Reviewer: ```suggestion
after upgrade tests pass. 
```

> and pre-production and production environments that participate in connected customer program about the stability of the release.

Reviewer: ```suggestion

```
No data being collected for OKD by default

or `stable-{product-version}` channels to the latest release in {product-version}.z,
although there can be delays while safe paths away from troubled releases are
constructed and verified.
endif::openshift-origin[]

ifdef::openshift-origin[]

> The presence of an update recommendation in the `stable-{product-version}`

Reviewer: ```suggestion
The presence of an update recommendation in the `stable-4`
```

channel is a declaration that the update is fully supported while it is in the
channel. While releases will never be removed from the channel, update recommendations
that exhibit serious issues will be removed from the channel. Updates initiated
after the update recommendation has been removed might not be supported.

> Red Hat will eventually provide supported update paths from any supported release in the

Reviewer: ```suggestion

```
OKD is not supported

Hat uses this program to observe the impact of updates on your specific hardware and software
configurations. Future releases might improve or alter the pace at which updates move
from the `fast-{product-version}` to the `stable-{product-version}` channel.
endif::openshift-origin[]

> ifdef::openshift-origin[]

Reviewer: Lets remove the whole section - we don't have a nice way to block the edges right now
